### DISCUSSION TITLE
MySqlDatabaseError, "Unknown column 'generation_expression' in 'field list'"

### DISCUSSION THREAD

**Discussion by heliping** [SeaQL/sea-orm#1225] [Q&A] [answered]
## Description

use cli to generate entity, report error.

## Steps to Reproduce

```
[help4@dev23 rhuax]$ sea-orm-cli generate entity -u mysql://root:sanysany@10.100.0.211:3306/huaxrobot -o entities2
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Database(MySqlDatabaseError { code: Some("42S22"), number: 1054, message: "Unknown column 'generation_expression' in 'field list'" })', /home/help4/.cargo/registry/src/github.com-1ecc6299db9ec823/sea-schema-0.10.2/src/mysql/discovery/executor/real.rs:29:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[help4@dev23 rhuax]$ 
```

## Versions
![image](https://user-images.githubusercontent.com/24326525/201580460-4cb680c8-942e-4ad8-943a-41f52f87df70.png)

**Comment by billy1624** [comment]
Hey @heliping, welcome and thanks for the filing.

I suspect this error originate from the incompatible MySQL version. I tested it on MySQL 5.7.36, and it was fine.

![image](https://user-images.githubusercontent.com/30400950/201568917-3b13b9ba-1870-4f46-8162-8d4262c2210e.png)

**Comment by billy1624** [comment]
I'll convert this as Q&A

**Comment by heliping** [comment]
so the solution is update mysql? 
my database already has many data in it, cannot to solve?

**Reply by billy1624** [reply]
No, it's not a must to upgrade MySQL. You need to define the entity by hand or wait for this patch to be released https://github.com/SeaQL/sea-schema/pull/86

**Comment by heliping** [comment]
only get error when generate the entity.
code the entity by hand, it works fine.

**Reply by billy1624** [reply]
The bug originated from sea-schema and it only affect sea-orm-cli

**Comment by billy1624** [comment]
Hey @heliping, this will be patched on the next sea-orm-cli release.
https://github.com/SeaQL/sea-schema/pull/86

**Comment by billy1624** [comment, marked as answer]
This bug has been patched on [sea-schema 0.10.3](https://github.com/SeaQL/sea-schema/releases/tag/0.10.3).
Please re-install `sea-orm-cli` by executing `cargo install sea-orm-cli --force` in order to generate entities on MySQL 5.6
